types/parse/index.d.ts

```diff
@@ -891,6 +891,7 @@ namespace Parse {
         logIn(options?: FullOptions): Promise<this>;
         authenticated(): boolean;
         isCurrent(): boolean;
+        verifyPassword(username?: string, password?: string, options?: UseMasterKeyOption): Promise<this>;
```

Comment:
This isn't the right instance method signature, according to [source code](https://github.com/parse-community/Parse-SDK-JS/blob/d9cc4d2c013df1c011e526cf2bc04fc91820eb47/src/ParseUser.js#L520).

```suggestion
        verifyPassword(password: string, options?: RequestOptions): Promise<this>;
```